builtin/providers/azurerm/encryption_settings.go

```diff
@@ -0,0 +1,163 @@
+package azurerm
+
+import (
+	"github.com/Azure/azure-sdk-for-go/arm/compute"
+	"github.com/Azure/azure-sdk-for-go/arm/disk"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func encryptionSettingsSchema() *schema.Schema {
```

Comment:
It's shared for both os disk and managed disks, which is why I pulled it out into its own separate file.  (If they diverge later we can diverge the code, but IMHO there's already too much duplicated code, esp. between VM and VMSS.